fvm/evm/stdlib/contract.cdc

```diff
@@ -16,15 +16,6 @@ contract EVM {
             self.bytes = bytes
         }
 
-        /// Deposits the given vault into the EVM account with the given address
-        access(all)
-        fun deposit(from: @FlowToken.Vault) {
-            InternalEVM.deposit(
-                from: <-from,
-                to: self.bytes
-            )
-        }
-
```

Comment:
Maybe also update the FLIP to reflect this change, both in the Cadence code, and also by adding a line or two that explains the rational (can just copy the description of the linked issue).

Reply:
the FLIP is updated with changes on the Cadence code side but I'll add a small explainer why its only available on the COAs.